### PR TITLE
Allow for ENV_PATH variable for OpenShift integration, add documentation (#1, #3)

### PR DESCRIPTION
**Pull request by ssciolla**
This PR tweaks how the `env.json` configuration file is loaded, moving the file's expected location to a `secrets` sub-directory inside the `config` directory and allowing for an `ENV_PATH` environment variable to be set in OpenShift to set the path (allowing for different file names, primarily). It also adds an application overview, configuration instructions, and OpenShift deployment details to `README.md`, as well as makes general formatting changes to the file. This PR aims to aims to resolve issue #1 and finish resolving issue #3.

### CODE REVIEW

**Comment by ssciolla** [comment]
@pushyamig , please let me know if the recent changes address your requests for more database documentation.

File: create_db.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
 logger = logging.getLogger(__name__)
 
 try:
-    with open(os.path.join('config', 'env.json')) as env_file:
+    config_path = os.getenv("ENV_PATH", os.path.join('config', 'secrets', 'env.json'))
```
Comment (zqian) [inline create_db.py]: nice move of env.json into the secrets folder.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,133 @@
...
+
+* By default, the application will run with the assumption that the JSON configuration file will be named `env.json`. 
+  However, `inventory.py` will also check for the environment variable `ENV_PATH`. This variable can be set using the 
+  OpenShift pod configuration file. To use a different name for the JSON file, set `ENV_PATH` to a path beginning with 
+  `config/secrets/` and ending with the file name. With this set, the `env` block in the `.yaml` will look 
+  something like this:
+  ```
+  - env:
+      - name: ENV_FILE
+        value: /config/secrets/env_test.json
```
Comment (zqian) [inline README.md]: env_test.json for the test server? Or should it be env.json

File: README.md [hunk 3]
```diff
@@ -1,40 +1,133 @@
...
+While performing any of the actions described below, use a terminal, text editor, or file utility as necessary. Some sample command-line instructions are provided for some steps.
+
+
+### Configuration
+
+To configure the application before installation and usage (see the next section), you must first perform a few steps, including the creation of a configuration file called `env.json`. Complete the following items in order.
+
+1. Clone and navigate into the repository.
+    ```
+    https://github.com/tl-its-umich-edu/course-inventory.git  # HTTPS
```
Comment (zqian) [inline README.md]: specify out the "git clone" command, since other code snippets are complete code lines.

File: README.md (hunk 2 above)
Comment (ssciolla) [reply README.md]: It's an example of re-naming the `env.json` file, so I used something else here. Is that okay?

File: README.md (hunk 3 above)
Comment (ssciolla) [reply README.md]: Wups

Comment (ssciolla) [reply README.md]: Fixed this.

**Comment by ssciolla** [comment]
I'm merging now, let me know if you want something else in the README going forward.

**Pull request merged**